### ISSUE TITLE
Computed Property Modal

### ISSUE THREAD

**Issue by velkymx** [ProcessMaker/screen-builder#345]
The save/cancel buttons in this modal don’t follow our standard UI patterns. They should be flipped and only say Save and Cancel.

Also the “Property Saved” alert persists even when I go to add another Computed Property, as seen above. This should show in the main window, not in the modal.

Also, the validation errors and red outlines appear in this modal as soon as it’s opened, contrary to all of our other modals.

In conclusion, this modal is quite something.

Current Modal: 

![image](https://user-images.githubusercontent.com/535300/62907875-319a9080-bd2a-11e9-96a4-0dc3956b8328.png)

Correct Modal Pattern: 

![image](https://user-images.githubusercontent.com/535300/62907905-4a0aab00-bd2a-11e9-8157-fd5beb6b6112.png)
